src/lib/modules/executor/executor.go

```diff
@@ -0,0 +1,105 @@
+package executor
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/bench-routes/bench-routes/tsdb/file"
+	"github.com/go-ping/ping"
+)
+
+type Ping struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Jitter struct {
+	Min time.Duration `json:"minVal"`
+	Max time.Duration `json:"maxVal"`
+	Avg time.Duration `json:"avgVal"`
+}
+
+type Monitor struct {
+	ResDelay  time.Duration `json:"resDelay"`
+	ResLength int64         `json:"resLength"`
+}
+
+// ExecuteMachine executes ping and jitter operations
+func ExecuteMachine(app file.Appendable, host string) error {
```

Comment:
```suggestion
// Machine evaluates calculations related to ping and jitter.
func Machine(app file.Appendable, host string) error {
```

Remember: When you have a package name as `X`, the naming of functions should be such that you can easily read the package_name + function_name and it makes sense.